ferrocene/doc/user-manual/src/cli.rst

```diff
@@ -241,11 +241,6 @@ Command-Line Interface
 
    .. cli:option:: -C linker=<path>
 
-      .. caution::
-
-         Only the paths listed in section Compilation targets are within the
-         scope of the Ferrocene |iso_ref| qualification.
```

Comment:
We should say this that it is possible to use it only for targets that require a linker driver, and in that case the linker driver must respect the option.

Reply:
an attempt https://github.com/ferrocene/ferrocene/pull/324/commits/74af1d982dd352c1ebc3f35521dd94df8f99f791